CHANGELOG.md

```diff
@@ -21,11 +21,15 @@ All notable changes to this project will be documented in this file.
   - `ClusterCondition::is_good`
   - `ClusterOperationsConditionBuilder::new`
   - `commons::pdb::default_pdb_enabled`
-
-[#674]: https://github.com/stackabletech/operator-rs/pull/674
+- Add interoperability between the `time` crate and the `stackable_operator::time::Duration` struct. This is opt-in and
+  requires the `time` feature to be enabled. Additionally, adds `Add`, `AddAssign`, `Sub`, and `SubAssign` operations
+  between `Duration` and `std::time::INstant`. Further adds a new helper dunction `Duration::now_utc` which calculates
```

Comment:
```suggestion
  between `Duration` and `std::time::Instant`. Further adds a new helper function `Duration::now_utc` which calculates
```

Reply:
Waaay ahead of you, :) I already fixed it in dbca142